jobs/integr8ly/installation-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,102 @@
+---
+
+- job:
+    name: installation-pipeline-qe-pony
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: CLUSTER_URL
+            description: "URL of cluster for installation."
+        - string:
+            name: GH_CLIENT_ID
+            description: "GitHub Client ID for OAuth Apps, required for some of the walkthroughs. Can be left empty"
+        - string:
+            name: GH_CLIENT_SECRET
+            description: "GitHub Client Secret for OAuth Apps, required for some of the walkthroughs. Can be left empty"
```

Comment:
Improvement idea: somebody could create GH App just for Pony and we could have it as default value for these params, or hard-coded all together.

Reply:
There can be used (hard-coded as defaults):
b15294d65592a5e6afb4
539a4a94cda8854d610d2dbef25881a31b49cba2

Reply:
Yeah why not...I'll update